airbyte-integrations/bases/source-acceptance-test/unit_tests/test_spec_unit.py

```diff
@@ -23,64 +23,77 @@
 #
 
 import io
-import os
-from pathlib import Path
 
 import docker
+import pytest
 from source_acceptance_test.utils import ConnectorRunner
 
 
+def create_dockerfile(text, tag, tmp_path):
+    client = docker.from_env()
+    fileobj = io.BytesIO()
+    fileobj.write(text)
+
+    fileobj.seek(0)
+    args = {
+        "fileobj": fileobj,
+        "tag": tag,
+        "forcerm": True,
+        "rm": True,
+    }
+    image, iterools_tee = client.images.build(**args)
+    docker_runner = ConnectorRunner(image_name=tag, volume=tmp_path)
+    return docker_runner
+
+
+@pytest.fixture
+def dockerfile_valid(tmp_path):
+    dockerfile_text = b"""
+        FROM scratch
+        ENV AIRBYTE_ENTRYPOINT "python /airbyte/integration_code/main.py"
+        ENTRYPOINT ["python", "/airbyte/integration_code/main.py"]
+        """
+    docker_runner = create_dockerfile(dockerfile_text, "my-valid-one", tmp_path)
+    return docker_runner
```

Comment:
we should yield `image.tag` from all fixtures, and call `remove(image)` after yield
everything after yield will be called right before pytest will exit from fixture scope (in this case test method)